MDAnalysisData/adk_equilibrium.py

```diff
@@ -5,88 +5,32 @@
 https://figshare.com/articles/Molecular_dynamics_trajectory_for_benchmarking_MDAnalysis/5108170/1
 """
 
-from os.path import dirname, exists, join
-from os import makedirs, remove
-import codecs
-
 import logging
 
-from .base import get_data_home
-from .base import _fetch_remote
-from .base import RemoteFileMetadata
-from .base import Bunch
+from .base import RemoteFileMetadata, Dataset
 
-NAME = "adk_equilibrium"
-DESCRIPTION = "adk_equilibrium.rst"
-# The original data can be found at the figshare URL.
-# The SHA256 checksum of the zip file changes with every download so we
-# cannot check its checksum. Instead we download individual files.
-# separately. The keys of this dict are also going to be the keys in the
-# Bunch that is returned.
-ARCHIVE = {
-    'topology': RemoteFileMetadata(
-        filename='adk4AKE.psf',
-        url='https://ndownloader.figshare.com/files/8672230',
-        checksum='1aa947d58fb41b6805dc1e7be4dbe65c6a8f4690f0bd7fc2ae03e7bd437085f4',
-    ),
-    'trajectory':  RemoteFileMetadata(
-        filename='1ake_007-nowater-core-dt240ps.dcd',
-        url='https://ndownloader.figshare.com/files/8672074',
-        checksum='598fcbcfcc425f6eafbe9997238320fcacc6a4613ecce061e1521732bab734bf',
-    ),
-}
 
 logger = logging.getLogger(__name__)
 
 
-def fetch_adk_equilibrium(data_home=None, download_if_missing=True):
-    """Load the AdK 1us equilibrium trajectory (without water)
-
-    Parameters
-    ----------
-    data_home : optional, default: None
-        Specify another download and cache folder for the datasets. By default
-        all MDAnalysisData data is stored in '~/MDAnalysis_data' subfolders.
-        This dataset is stored in ``<data_home>/adk_equilibrium``.
-    download_if_missing : optional, default=True
-        If ``False``, raise a :exc:`IOError` if the data is not locally available
-        instead of trying to download the data from the source site.
-
-    Returns
-    -------
-    dataset : dict-like object with the following attributes:
```

Comment:
and another thing lost is the clear description of what the returned Bunch will have....

Reply:
I don't want to loose this...